tools/component-builder/index.js

```diff
@@ -23,7 +23,7 @@ function clean() {
 const build = gulp.series(
   clean,
   gulp.parallel(
-    css.buildCSS,
+    css.buildVars,
```

Comment:
The old, legacy build tasks are still in the code, but instead of calling them, we run the modern vars build tasks.